oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+Note that this section is GitHub-specific because that's where the Open edX platform's code resides.  The idea of "issues" and "pull requests" is otherwise portable to other version control platforms.
+
+GitHub Issues (Optional)
+************************
+
+We would encourage decision makers to use a GitHub Issue to have initial conversations around the context of a decision, or to write down upcoming/potential decisions. It may be that - at the start of a project - we're not even sure what decisions need to be made.  They may not even reveal themselves until we're in the middle of coding.
+
+An Issue is a great place to have these initial, loose, conversations about a decision space without the burden of creating a full ADR (creating a new document in the right place, authoring the sections of an ADR, creating a branch, committing, creating a PR, etc.)  It may not be necessary for more well-defined or focused decisions.  We want to preserve this as an optional first step, as we recognize that creating a "formal" ADR can feel inexplicably daunting or like a distraction, especially when possible decisions come up in the course of working on an implementation.  The mental burden of breaking "flow state" and creating a new ADR document is often problematic.  We hope jotting some notes in an Issue as a starting point will allow engineers to record their potential decision point and quickly continue their work.
+
+GitHub Pull Requests
+********************
+
+Whether or not a GitHub Issue is used, a GitHub Pull Request will exist for creation/merging of the ADR.  This is the primary medium for the Advisory Process, and where we expect conversations will take place.  By the time a pull request is created, we expect the decision maker will have written down the context of the decision - necessary to orient advisors to the problem space - as well as draft of their potential decision.
+
+Decision Maker's Checklist
+--------------------------
+
+In order to help ADR authors ensure they get advice from the proper set of stakeholders, we will create a checklist of possible groups/individuals to choose from.  It's the responsibility of the author to ensure the relevant people are aware of the ADR and are given the opportunity to provide feedback.
+
+The Decision Maker's Checklist is a document that a decision authors can review to ensure they're seeking advice from the proper stakeholders and experts when authoring an ADR. It's that individual's responsibility to select all those stakeholders and experts (which may be individuals or groups) that are impacted by their decision based on their understanding of the scope.  Other individuals or groups may be added later as the scope and impact is better understood.
+
+GitHub Groups
```

Comment:
fwiw they're called Teams in GH, not sure that matters.

Reply:
Ah!  Yeah, let's get that right since this is written in a GitHub-specific way.